core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -37,18 +38,20 @@
 
     public static final String DEFAULT_CONFIG_FILE_NAME = "mobile-services.json";
 
-    private static final String TAG = "AEROGEAR/CORE";
     private static final int DEFAULT_READ_TIMEOUT = 30;
     private static final int DEFAULT_CONNECT_TIMEOUT = 10;
     private static final int DEFAULT_WRITE_TIMEOUT = 10;
+
+    @SuppressLint("StaticFieldLeak")
+    private static MobileCore instance;
+    // TODO Move to instance
```

Comment:
Maybe we could add ticket, but it seems to be trivial task.

Reply:
Not so trivial because of the tests, that is why I wanna do it on a separate PR